spec/workers/virus_scan_worker_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe VirusScanWorker, type: :worker do
+  let(:worker) { described_class.new }
+  let(:asset) { FactoryGirl.create(:asset) }
+
+  it "calls out to the VirusScanner to scan the file" do
+    scanner = double("VirusScanner")
+    expect(VirusScanner).to receive(:new).with(asset.file.path).and_return(scanner)
```

Comment:
Minor: I would move the first of these lines into a `let(:scanner)` statement and use an `instance_double` of `VirusScanner`. And I'd move the second line into a `before` block and convert the `expect` into an `allow`.

Reply:
I've just realised that you've moved these examples from the `Asset` spec, so you can ignore my previous comment and I will refrain from making more suggestions for improvement in this commit.